src/API.jl

```diff
@@ -23,22 +23,23 @@ function dependencies(ctx::Context)::Dict{UUID, PackageInfo}
     find_registered!(ctx, UUID[pkg.uuid for pkg in pkgs])
     return Dict(pkg.uuid => Operations.package_info(ctx, pkg) for pkg in pkgs)
 end
-
-project() = project(Context())
-function project(ctx::Context)
-    ctx.env.pkg !== nothing || pkgerror("Active environment is not a project.")
-    return ProjectInfo(
-        name         = ctx.env.pkg.name,
-        uuid         = ctx.env.pkg.uuid,
-        version      = ctx.env.pkg.version,
+dependencies(fn::Function, uuid::UUID) = fn(dependencies()[uuid])
+
+environment() = environment(Context())
+function environment(ctx::Context)::EnvironmentInfo
```

Comment:
I think this should still be called "project"? AFAIU, environment includes the stack of projects while this only describes a specific project.

Reply:
Ah yes, I was using different terminology in my head :grimacing:. I'll change it back